rollbar-spring-webmvc/src/main/java/com/rollbar/spring/webmvc/RollbarWebApplicationInitializer.java

```diff
@@ -0,0 +1,23 @@
+package com.rollbar.spring.webmvc;
+
+import com.rollbar.web.filter.RollbarFilter;
+import com.rollbar.web.listener.RollbarRequestListener;
+import javax.servlet.ServletContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.WebApplicationInitializer;
+
+public class RollbarWebApplicationInitializer implements WebApplicationInitializer {
+
+  @Override
+  @Autowired
```

Comment:
I think we don't need this annotation here.

Reply:
done